docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,33 +19,42 @@
   },
```

Comment:
<p>This is great, thank you!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Thanks!</p>